### PR TITLE
Separate LR scheduler config

### PR DESCRIPTION
**Pull request by jemrobinson**
Separate LR scheduler config (which controls how Lightning logs learning rate to e.g. W&B) from the options which control scheduler behaviour (e.g. eta_min).

### CODE REVIEW

File: icenet_mp/models/base_model.py [hunk 1]
```diff
@@ -42,6 +46,7 @@ def __init__(  # noqa: PLR0913
         latitudes_fn: Callable[[], dict[str, list[float]]] | None = None,
         longitudes_fn: Callable[[], dict[str, list[float]]] | None = None,
         loss: DictConfig,
+        lr_scheduler: DictConfig,
         metrics: list[str] | None = None,
```
[the remark this answers is not in the document]
Comment (jemrobinson) [reply icenet_mp/models/base_model.py:50]: No, this is deliberate.
